driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -358,8 +358,12 @@ private void stopConfiguration() throws Exception {
                 server.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
             }
             for (ClientBootstrapResolver clientResolver : configuration.getClientResolvers()) {
-                ClientBootstrap client = clientResolver.resolve();
-                client.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
+                try {
+                    ClientBootstrap client = clientResolver.resolve();
+                    client.setPipelineFactory(pipelineFactory(pipeline(closeOnExceptionHandler)));
+                } catch (RuntimeException e) {
```

Comment:
> why does this throw an exception ?
Because the clientResolver resolves properties ${connectURI} that might not be set.  This is essentially one last kick for K3po to run the connects when they aren't explicitly called so we can return a correct diff.  IF the properties in the handlers are not resolved this throws a hard exception and shuts down the control channel, which isn't good. I'd be ok with adding a try catch to the serverResolve.resolve as well...